server/index.js

```diff
@@ -2,13 +2,13 @@ const express = require('express');
 const path = require('path');
 const router = require('./router.js');
 const port = 3000;
-let app = express();
+
+const app = express();
 
 app.use(express.json());
 app.use(express.static(path.join(__dirname, '../client/dist')))
-// app.get('/home')
-app.use('/api', router)
```

Comment:
Any reason why you're using `/routes` instead of `/api`?

Reply:
'/api' was a test endpoint when we first set up index.js. I believe the current endpoint is 
https://app-hrsei-api.herokuapp.com/api/fec2/:CAMPUS_CODE/, so we shouldn't need to use '/api' as the default route endpoint, and '/' should suffice?